pages/linux/bzip2.md

```diff
@@ -0,0 +1,15 @@
+# bzip2
+
+> A block-sorting file compressor.
+
+- Compress file:
+
+`bzip2 {{path/to/file_to_compress}}`
+
+- Decompress file:
+
+`bzip2 -d {{path/to/compressed_file.bz2}}`
+
+- Decompress to STDOUT:
+
+`bzip2 -dc {{path/to/compressed_file.bz2}}`
```

Comment:
`-dc` nicely fits into 'decompress to console' indeed :dart: 
Do you want me to change command description ?